src/compiler/checker.ts

```diff
@@ -50760,7 +50760,7 @@ export function signatureHasRestParameter(s: Signature) {
 }
 
 /** @internal */

```

Comment:
This reminds me that our jsdoc lint rule should complain about this doc on unexported declarations.

Reply:
(I did this)